### PR TITLE
Login+signup styling

### PR DESCRIPTION
**Pull request by Saskiazinha**

### CODE REVIEW

File: frontend/src/security/LoginPage.js [hunk 1]
```diff
@@ -46,3 +49,43 @@ export default function LoginPage() {
...
+  height: 100vh;
+
+  h1 {
+    text-align: center;
+  }
+`;
+
+const FormStyling = styled.form`
+  display: grid;
+  grid-template-rows: 50px 50px 50px 60px 50px;
```
Comment (HennyHaelbich) [inline frontend/src/security/LoginPage.js:64]: bei ist glaube ich fr die bessere Einheit

**Comment by HennyHaelbich** [comment]
LGTM

File: frontend/src/GlobalStyle.js [hunk 2]
```diff
@@ -25,9 +25,16 @@ export default createGlobalStyle`
   
   html, body {
     margin: 0;
-     font-size: 112.5%;
+     font-size: 1rem;
```
Comment (Asamya) [inline frontend/src/GlobalStyle.js:28]: Do you need a relativ font-size in the body element?

File: frontend/src/security/LoginPage.js (hunk 1 above)
Comment (Asamya) [inline frontend/src/security/LoginPage.js:64]: Try to use `fr` or % to keep the responsiveness later on.

File: frontend/src/security/SignUpPage.js [hunk 3]
```diff
@@ -90,22 +99,64 @@ export default function SignUpPage() {
...
     if (!/[A-Z]/.test(password1)) {
-      throw new Error("Password must contain at least one uppercase letter.");
+      throw new Error("Password must contain at least one uppercase letter!");
     }
   }
 }
+
+const SignUpStyling = styled.div`
+  background-color: var(--turquoise-main);
+  height: 100vh;
```
Comment (Asamya) [inline frontend/src/security/SignUpPage.js:124]: If you use the color on `100vh` on more than one page try to style this in App.js

File: frontend/src/security/SignUpPage.js [hunk 4]
```diff
@@ -90,22 +99,64 @@ export default function SignUpPage() {
...
+
+  h1 {
+    text-align: center;
+  }
+`;
+
+const FormStyling = styled.form`
+  display: grid;
+  grid-template-rows: 50px 50px 50px 50px 50px 70px;
```
Comment (Asamya) [inline frontend/src/security/SignUpPage.js:133]: Maybe you can try a more responsive design?

**Review by Asamya — APPROVED** [review]
Really good work on the styling only some minor comments. Keep up the good work.

**Pull request merged**